spec/koans.js

```diff
@@ -42,39 +43,39 @@ describe('`const` is like `let` plus read-only. ', () => {
 
   describe('scalar values are read-only', () => {
     it('number are read-only', () => {
-      // const constNum = 0;
-      // constNum = 1;
+      const constNum = 0;
+      let constNum = 1;
 
-      //expect(constNum).toBe(0);
+      expect(constNum).toBe(0);
     });
 
     it('string are read-only', () => {
-      // const constString = "I am a const";
-      // constString = "Cant change you?";
+      const constString = "I am a const";
+      let constString = "Cant change you?";
```

Comment:
Same here, you can't reassign a const, so you can just leave this one commented out!